config.go

```diff
@@ -117,6 +117,16 @@ type RuntimeConfig interface {
 	//	foo := wazero.NewRuntimeWithConfig(context.Background(), config)
 	// 	bar := wazero.NewRuntimeWithConfig(context.Background(), config)
 	WithCompilationCache(CompilationCache) RuntimeConfig
+
+	// WithCustomSections toggles parsing of "custom sections". Defaults to false.
+	//
+	// When enabled, it is possible to retrieve custom sections from a CompiledModule:
+	//
+	//	config := wazero.NewRuntimeConfig().WithCustomSections(true)
+	//	r := wazero.NewRuntimeWithConfig(ctx, config)
+	//  c, err := r.CompileModule(ctx, wasm)
+	//  customSections := c.CustomSections()
```

Comment:
@evacchi you closely compare these lines, you mixed spaces and tabs, that's what I meant the previous comment.

Reply:
oh my! :D 

![](https://media.giphy.com/media/w0vFxYaCcvvJm/giphy.gif)

Reply:
fix pushed!